lib/local_authorities_importer.rb

```diff
@@ -0,0 +1,72 @@
+require 'json_downloader'
+
+class LocalAuthoritiesImporter
```

Comment:
It would be nice to start out with an `import` module - this PR adds 3 importers and we know we need to write more (for services, interactions and links), so we can keep them clearly separated from other lib code. [Transition does this](https://github.com/alphagov/transition/tree/master/lib/transition/import) and it similarly imports data from various sources.